sndev

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+sndev__start() {
+  if [ ! -x "$(command -v docker-compose)" ]; then
+    echo "docker compose is not installed"
+    echo "installation instructions are here: https://docs.docker.com/desktop/"
+    exit 0
+  fi
+
+  if ! [ -f .env.sndev ]; then
+    echo ".env.sndev does not exist."
+    echo "creating from .env.sample"
+    cp .env.sample .env.sndev
+  fi
+
+  echo "Starting application"
+  docker compose up
```

Comment:
lol, I didn't know you can run `docker compose`. I always ran `docker-compose`.

I hope that's not platform dependent. But seems to be compatible between macOS and arch-based systems and we don't talk about Windows, I guess.

Reply:
I'd guess it's just syntactic sugar